app/views/layouts/_admin_sidebar.html.haml

```diff
@@ -116,7 +116,11 @@
         - if can? :update, @conference.tickets.build
           %li{class: active_nav_li(admin_conference_tickets_path(@conference.short_title)) }
             = link_to 'Tickets', admin_conference_tickets_path(@conference.short_title)
-
+  - if can? :manage, @conference.booths
```

Comment:
This cannot work obviously, try this with a user that is **not** an admin, and see if this ever appears. 
The `can?` statement needs either a model, or a record. You are supplying it with a collection of objects (`@conference.booths`), and this cannot be evaluated to true. 

Go back to the documentation of CanCanCan and check the correct syntax. Where does it say you can supply a collection?